### ISSUE TITLE
SplitButton component does not work for JAWS screen reader

### ISSUE THREAD

**Issue by aubreyquinn** [microsoft/fluentui#19576]
### Environment Information

- **Package version(s)**: 0.54.0
- **Browser and OS versions**: Edge & Windows 10

### Describe the issue:

When we navigate to the SplitButton component with the JAWS screen reader, it announces "push Alt + down" but when you push the "Alt + down" controls on the keyboard, it does nothing. It seems to work for NVDA reader.

### Please provide a reproduction of the issue in a codepen:

Code sandbox: https://codesandbox.io/s/fluent-ui-example-splitbutton-su8t8

#### Actual behavior:

Nothing opens.

#### Expected behavior:

Should open the menu when "Alt + down key" are pressed.

### Documentation describing expected behavior

https://fluentsite.z22.web.core.windows.net/0.54.0/components/split-button/definition

**Comment by jurokapsiar** [comment]
Alt+Down is a shortcut that JAWS is using in virtual cursor mode. To fully interact with the components, it is necessary to switch to forms mode. Node: depending on the user settings, JAWS can be set to auto mode upon start and its mode might switch automatically.

Verified that alt+down works when user switches to forms mode (insert+z in the laptop mode).
In virtual cursor mode, user can navigate to both parts of the split button and simulate click on them using arrow keys/enter.

Closing this as by design.
